### ISSUE TITLE
Improvement : Firefox GNOME Tutorial

### ISSUE THREAD

**Issue by sbaconnais** [chpii/Headerbar#20]
You can install this stylish extension : https://userstyles.org/styles/107715/hide-australis-back-forward-buttons

And this extension : https://addons.mozilla.org/fr/firefox/addon/back-forward-separate/?src=userprofile

To get the new tab icon in the right place.

![capture du 2015-10-28 15-08-59](https://cloud.githubusercontent.com/assets/5074414/10790852/e88acfa0-7d85-11e5-83da-d93f677a3ad3.png)

**Comment by DoctorJellyface** [comment]
Can't this be achieved by a single Stylish extension?

**Comment by chpii** [comment]
Navigation buttons are part of urlbar-container, I don't know a way to move them outside.

This extension adds custom navigation buttons that can be moved into toolbar. That userstyle hides original ones.

**Comment by DoctorJellyface** [comment]
Yeah, I understand how it works, was just wondering if there wasn't an easier way.
